src/llvmir2hll/ir/module.cpp

```diff
@@ -374,6 +374,30 @@ bool Module::hasFuncDefinitions() const {
 	);
 }
 
+/**
+* @brief Are there any decompiler-defined functions in the module?
+*/
+bool Module::hasDecompilerDefinedFuncs() const {
+	for (auto &func : funcs) {
+		if (config->isDecompilerDefinedFunc(func->getInitialName())) {
+			return true;
+		}
+	}
+	return false;
+}
+
+/**
+* @brief Returns all decompiler-defined functions in the module.
+*/
+FuncSet Module::getDecompilerDefinedFuncs() const {
+	return getFuncsSatisfyingPredicate(
+		[this](auto func) {
+			return config->isDecompilerDefinedFunc(func->getInitialName());
+		}
+	);
+}
+
+
```

Comment:
Nitpick: Too many newlines (one is enough).

Reply:
Hm, I would swear I removed it from the commit. Nevertheless, thank you I'll fix this.